src/proto/grpc/testing/messages.proto

```diff
@@ -103,9 +103,6 @@ message SimpleRequest {
 
   // If set the server should record this metrics report data for the current RPC.
   TestOrcaReport orca_per_query_report = 11;
-
```

Comment:
So the source of this file is really https://github.com/grpc/grpc-proto/blob/master/grpc/testing/messages.proto no?

I see that changes from that file are merged in here and we are also making new changes here. Will they then be merged into the file in grpc-proto? What is the normal process?

Reply:
Yes. I will make a PR in that repo if we agree on these changes. It was easier for me to do the changes in this repo first so I could see how it works with the actual code.